configs/model/fly_tethered.yaml

```
@@ -0,0 +1,180 @@
+N_FRAMES_PER_CLIP: 300
+
+# Tolerance for the optimizations of the full model, limb, and root. 
+# TODO: Re-implement optimizer loops to use these tolerances
+FTOL: 5.0e-03
+ROOT_FTOL: 1.0e-05
+LIMB_FTOL: 1.0e-06
+
+# Number of alternating pose and offset optimization rounds. 
+N_ITERS: 6
+
+KP_NAMES: 
```

yamllint

[error] trailing spaces

(trailing-spaces)